rootfs-builder/README.md

```diff
@@ -124,3 +85,88 @@ must be met:
    $ # build image based rootfs created above
    $ ./image-builder/image_builder.sh "${PWD}/fedora_rootfs"
    ```
+
+## Adding support for a new guest OS
+
+The `rootfs.sh` script will check for immediate sub-directories
+containing the following expected files:
+
+- A `bash(1)` script called `rootfs_lib.sh`
+
+  This file must contain a function called `build_rootfs()` which
```

Comment:
Lines 96-98 suggested rewrite: This file must contain a function called `build_rootfs()`, which must receive the path to where the rootfs is created, as its first argument.